get-shit-done/workflows/reapply-patches.md

```
@@ -269,30 +269,51 @@ After writing each merged file, verify that user modifications survived the merg
+VERIFY_OUTPUT="$(node "${GSD_HOME}/scripts/verify-reapply-patches.cjs" \
+  --patches-dir "$PATCHES_DIR" \
+  --config-dir "$CONFIG_DIR" \
+  $PRISTINE_FLAG \
+  --json 2>&1)"
+VERIFY_STATUS=$?
 ```
 
-**If any row in the Hunk Verification Table shows `verified: no`**, STOP and report to the user:
+**If `VERIFY_STATUS` is non-zero**, STOP and report to the user, parsing the JSON output:
+
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)